pkgs/development/interpreters/bats/libraries.nix

```diff
@@ -47,6 +47,29 @@
     };
   };
 
+  bats-detik = stdenv.mkDerivation rec {
+    pname = "bats-detik";
+    version = "1.2.1";
+    src = fetchFromGitHub {
+      owner = "bats-core";
+      repo = "bats-detik";
+      rev = "v${version}";
+      sha256 = "sha256-2BEIqRSc21oPjd9BgTLg5mGyAdNJYA2b7gZe7Nj2dks=";
+    };
+    dontBuild = true;
+    installPhase = ''
+      mkdir -p "$out/share/bats/bats-detik"
+      cp -r lib/* "$out/share/bats/bats-detik"
+    '';
+    meta = {
+      description = "A library to ease e2e tests of applications in K8s environments";
+      platforms = lib.platforms.all;
+      homepage = "https://github.com/bats-core/bats-detik";
+      license = lib.licenses.cc0;
+      maintainers = with lib.maintainers; [ brokenpip3 ];
```

Comment:
Have you added yourself under maintainers file?

Reply:
done https://github.com/NixOS/nixpkgs/pull/265652/commits/227e6cdb9dd6214c88c15b68d33dfe71a6209b02, thanks!